common/static/js/vendor/jquery.leanModal.js

```diff
@@ -0,0 +1,61 @@
+// leanModal v1.1 by Ray Stone - http://finelysliced.com.au
```

Comment:
@andy-armstrong  I had the same concern and wanted to ask you about that.   Where would I need to include it?   (Maybe this is documented some place I haven't found?)